pkgs/top-level/python-packages.nix

```diff
@@ -11199,6 +11199,8 @@ self: super: with self; {
     cudnnSupport = false;
   };
 
+  thefuzz = callPackage ../development/python-modules/thefuzz { };
```

Comment:
Being such a Python package, I would suggest the Python's `callPackage`.
Hovever I am a bit lost on this.